src/main/java/core/basesyntax/shape/RightTriangle.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.shape;
+
+import core.basesyntax.type.Color;
+
+public class RightTriangle extends Shape {
+    private double base1;
+    private double base2;
+
+    public RightTriangle(Color color, double base1, double base2) {
+        super(color);
+        this.base1 = base1;
+        this.base2 = base2;
+    }
+
+    @Override
+    public double getArea() {
+        return (this.base1 * this.base2) / 2.0;
```

Comment:
Change the formula from '(this.base1 * this.base2) / 2.0;' to '(this.leg1 * this.leg2) / 2.0;' to match the updated variable names.